src/main/resources/db/migration/0250/V277__DeliverableDueDates.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE accelerator.deliverable_project_due_dates (
+    deliverable_id BIGINT NOT NULL REFERENCES accelerator.deliverables ON DELETE CASCADE,
+    project_id BIGINT NOT NULL REFERENCES projects ON DELETE CASCADE,
+    due_date DATE NOT NULL,
+    PRIMARY KEY (deliverable_id, project_id)
+);
+
+CREATE TABLE accelerator.deliverable_cohort_due_dates (
+    deliverable_id BIGINT NOT NULL REFERENCES accelerator.deliverables ON DELETE CASCADE,
+    cohort_id BIGINT NOT NULL REFERENCES accelerator.cohorts ON DELETE CASCADE,
+    due_date DATE NOT NULL,
+    PRIMARY KEY (deliverable_id, cohort_id)
+);
+
```

Comment:
We may end up also wanting indexes with the columns the other way around, but we can look at the query plans and see if extra indexes would be helpful once we're running real queries against these.